packages/by-name/imagepuller-eval/package.nix

```diff
@@ -0,0 +1,42 @@
+# Copyright 2025 Edgeless Systems GmbH
+# SPDX-License-Identifier: BUSL-1.1
+
+{ lib, buildGoModule }:
+
+buildGoModule (finalAttrs: {
+  pname = "imagepuller-eval";
+  version = "0.1.0";
+
+  # The source of the main module of this repo. We filter for Go files so that
+  # changes in the other parts of this repo don't trigger a rebuild.
+  src =
+    let
+      inherit (lib) fileset path;
+      root = ../../../.;
+    in
+    fileset.toSource {
+      inherit root;
+      fileset = fileset.unions [
+        (path.append root "imagepuller/go.mod")
+        (path.append root "imagepuller/go.sum")
+        (path.append root "imagepuller/client")
+        (path.append root "imagepuller/internal/api")
+        (path.append root "tools/imagepuller-eval")
+      ];
+    };
+
+  proxyVendor = true;
+  vendorHash = "sha256-7lo4SazUC52kSUgbZBE1lFMTPk1oOp+gRwJ0jSboO0k=";
+
+  sourceRoot = "${finalAttrs.src.name}/tools/imagepuller-eval";
+
+  env.CGO_ENABLED = 0;
+
+  ldflags = [ "-s" ];
+
+  preCheck = ''
+    export CGO_ENABLED=1
+  '';
```

Comment:
rather `doCheck = false`? imagepuller-eval has no tests, and imagepuller tests are already executed elsewhere.